### PR TITLE
[Blogging prompts v1] Add "Learn more" to "My Site" prompt card

### PR DESCRIPTION
**Pull request by RenanLukas**
Fixes #

To test:

- Enable blogging prompts feature flag;
- Tap on "My Site" in bottom navigation;
- Tap on "HOME" tab;
- Scroll to prompt card;
- Tap on overflow menu;
- Tap on "Learn more" button. Blogging prompts introduction should be shown.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and updated unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated `BloggingPromptsCardAnalyticsTrackerTest.kt`

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloggingprompts/BloggingPromptCardViewHolder.kt [hunk 1]
```diff
@@ -90,6 +96,14 @@ class BloggingPromptCardViewHolder(
                 R.id.view_more -> bloggingPromptsCardAnalyticsTracker.trackMySiteCardMenuViewMorePromptsClicked()
                 R.id.skip -> bloggingPromptsCardAnalyticsTracker.trackMySiteCardMenuSkipThisPromptClicked()
                 R.id.remove -> bloggingPromptsCardAnalyticsTracker.trackMySiteCardMenuRemoveFromDashboardClicked()
+                R.id.learn_more -> {
+                    bloggingPromptsCardAnalyticsTracker.trackMySiteCardMenuLearnMoreClicked()
+                    bloggingPromptCardMenu.context.startActivity(
+                        ActivityLauncher.createMainActivityAndShowBloggingPromptsOnboardingActivityIntent(
```
Comment (khaykov) [inline WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloggingprompts/BloggingPromptCardViewHolder.kt]: I'm not sure restarting the main activity is the best way to show the dialog, to be honest. And we should not do it from ViewHolder, which should not know about activity launcher and any static methods. Ideally host fragment would show a dialog though some sort of listener-LiveData combo.

Comment (RenanLukas) [reply WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloggingprompts/BloggingPromptCardViewHolder.kt]: Good catch! It definitely makes sense.
I've pushed a commit trying to address it 👍

**Review by develric — APPROVED** [review]
Code LGTM and works as described, approving 👍 !  

Not directly related to this PR, but wanted to double-check the following: in the modal, the prompt card example is showing the same avatar three times; is it a temporary state?

image

Since it's behind a feature flag, feel free to merge this one regardless and in case the above makes sense track/address it later 🙇 .

**Comment by khaykov** [comment]
@develric I fixed the avatars in a different PR :)

**Pull request merged**